module3/assessments/week5-1422.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Week 5 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_6490)
```

Comment:
Looks like this is part of your other PR, don't think it hurts to have this here but might cause merge conflicts?